### ISSUE TITLE
Signika specimen site rendering notdef for user

### ISSUE THREAD

**Issue by kontur** [google/fonts#3349]
A user reported to me that for them the [Signika specimen](https://fonts.google.com/specimen/Signika) loads like this on Mac OS 10.11.6 on latest Firefox and Safari:

![Screen Shot 2021-04-26 at 10 38 03](https://user-images.githubusercontent.com/1893577/116509430-f0fef400-a8cb-11eb-9e22-27ac3993e371.png)
![Screen Shot 2021-04-26 at 20 16 59](https://user-images.githubusercontent.com/1893577/116509433-f2302100-a8cb-11eb-9abf-998040ebc89b.png)

Apparently other specimens work, and the Signika specimen also works on Chrome. The text in the input should be supported by the font.

Is this some kind of know caching issue or something you have encountered before?

**Comment by mdisseny** [comment]
I confirm the issue on 10.11.6. While loading Signika with Lato, Lato displays fine while Signika doesn't, just blank space.

**Comment by kontur** [comment]
@mdisseny — are you referring to loading the two fonts on _some other_ site and observing this?
10.11.6 presumably being your Mac OS version, right? What browser did you use?

**Comment by mdisseny** [comment]
Exactly, excuse me for not being more specific. The OS is 10.11.6 El Capitan. The Firefox version is 78.11.0esr and Safari's 11.1.2. If I browse the Signika page on Google Fonts (https://fonts.google.com/specimen/Signika) I see exactly the same on your screenshots. The comment about Lato was to remark that there is no problem with the code to embed the fonts on the site as Lato works well on the site.

**Comment by kontur** [comment]
Super, thanks for confirming with more detail. Also good to know this is happening _outside_ of the GF specimen page!

@m4rc1e You ever encounter something like that with other fonts?
